sonic_platform_base/sonic_xcvr/xcvr_api_factory.py

```diff
@@ -36,15 +40,38 @@ def _get_id(self):
             return None
         return id_byte_raw[0]
 
+    def _get_vendor_name(self):
+       name_data = self.reader(129, 16)
```

Comment:
@AnoopKamath could you define 129 and 16 in a string like

VENDOR_NAME_OFFSET
VENDOR_NAME_LENGTH

Reply:
Taken care